### PR TITLE
fix: [ANDROAPP-3805]Fix analytics build warnings

### PR DESCRIPTION
**Pull request by Balcan**
## Description
**IMPORTANT** DO NOT REVIEW OR MERGE UNTIL DEVELOP HAS BEEN UPDATED WITH 2.4 BRANCH

[ jira issue ](https://jira.dhis2.org/browse/ANDROAPP-3805)

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Review by ferdyrod — APPROVED** [review]

**Comment by JaimeToca** [comment]
:+1:  let's wait until we merge the release branch to develop

**Pull request merged**
